test/ios_test_runner_unit_test.sh

```diff
@@ -446,6 +446,32 @@ function test_ios_unit_test_with_multi_filter() {
   expect_log "Executed 2 tests, with 0 failures"
 }
 
+function test_ios_unit_test_with_skip_filter() {
+  create_sim_runners
+  create_ios_unit_tests
+  do_ios_test --test_filter=-PassingUnitTest/testPass //ios:PassingUnitTest || fail "should pass"
+
+  expect_not_log "Test Case '-\[PassingUnitTest testPass\]' passed"
+  expect_log "Test Case '-\[PassingUnitTest testPass2\]' passed"
+  expect_log "Test Case '-\[PassingUnitTest testPass3\]' passed"
+  expect_log "Test Suite 'PassingUnitTest' passed"
+  expect_log "Test Suite 'PassingUnitTest.xctest' passed"
+  expect_log "Executed 1 test, with 0 failures"
+}
+
+function test_ios_unit_test_with_multi_skip_filter() {
+  create_sim_runners
+  create_ios_unit_tests
+  do_ios_test --test_filter=-PassingUnitTest/testPass,-PassingUnitTest/testPass2 //ios:PassingUnitTest || fail "should pass"
```

Comment:
can you add a case where we have both added and removed in the same arg? maybe include the class but exclude a single case from it?

Reply:
ah good call! I'll add that